app/code/Magento/SalesRule/Test/Unit/Model/ValidatorTest.php

```diff
@@ -368,11 +371,21 @@ public function testInitTotalsCanApplyDiscount()
         $validator->expects($this->at(1))->method('isValid')->with($item2)->willReturn(true);
 
         $item1->expects($this->any())->method('getParentItemId')->willReturn(false);
+        $item1->expects($this->any())->method('getParentItem')->willReturn(false);
```

Comment:
Does it really return false? I believe it usually returns null

Reply:
I was following the format used for `getParentItemId` on the line above, which should be returning an int according to the api, but will change to be null, as you're right that this is what it would be returning.